packages/test-synthetix/test.sh

```diff
@@ -0,0 +1,6 @@
+git clone git@github.com:Synthetixio/synthetix.git
```

Comment:
`git clone --depth=1 --branch develop https://github.com/Synthetixio/synthetix.git`

Reply:
Then you can remove the `checkout` step